im.riot.Riot.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "im.riot.Riot",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "riot",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--filesystem=xdg-run/keyring"
+    ],
+    "modules": [
+        {
+            "name": "riot",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x riot-web_*.deb",
+                "rm -f riot-web_*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
```

Comment:
Minor detail but you can remove these `rm` calls, since this isn't using `extra-data` the build dir containing build files is fine.

Reply:
Done